howso/utilities/features.py

```diff
@@ -172,8 +183,9 @@ def deserialize(
         ----------
         data : list of list or list of dict
             The context data.
-        columns : list of str
-            The case column mapping.
+        columns : Iterable of str, default None
```

Comment:
Default None is incorrect. This is not an optional parameter. If it isn't provided it is a runtime error.